view/two-column/public/inqlude.css

```diff
@@ -22,106 +21,157 @@ a:active {
   text-decoration: underline;
 }
 
-.ribbon {
-  background-color: #a00;
-  overflow: hidden;
-  position: absolute;
-  left: -2em;
-  top: 1em;
-  -moz-transform: rotate(-45deg);
-  -webkit-transform: rotate(-45deg);
-  -moz-box-shadow: 0 0 1em #888;
-  -webkit-box-shadow: 0 0 1em #888;
-}
-.ribbon a {
-  border: 1px solid #faa;
-  color: #fff;
-  display: block;
-  font: bold 81.25% 'Helvetiva Neue', Helvetica, Arial, sans-serif;
-  margin: 0.05em 0 0.075em 0;
-  padding: 0.5em 3.5em;
-  text-align: center;
-  text-decoration: none;
-  text-shadow: 0 0 0.5em #444;
-}
-
```

Comment:
It would be nice to keep the ribbon. We are still not there yet. There is a roadmap to get to stable. So I would prefer to use that. Or do you have a better idea?